source/reference/minio-mc/mc-license-info.rst

```diff
@@ -0,0 +1,71 @@
+===================
+``mc license info``
+===================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+.. mc:: mc license info
+
+Description
+-----------
+
+The :mc-cmd:`mc license info` command displays information about the cluster's license.
+
+For example, for a cluster not registered with |SUBNET|, the command returns information about the AGPLv3 license similar to the following:
+
+.. code-block:: shell
+
+   You are using GNU AFFERO GENERAL PUBLIC LICENSE Verson 3 (https://www.gnu.org/licenses/agpl-3.0.txt)
+
+   If you are building proprietary applications, you may want to choose the commercial license 
+   included as part of the Standard and Enterprise subscription plans. (https://min.io/signup?ref=mc)
+
+   Applications must otherwise comply with all the GNU AGPLv3 License & Trademark obligations.
+
+Examples
+--------
+
+Display the Current License for a Cluster with Alias ``minio1``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: shell
+   :class: copyable
+
+   mc license info minio1
+
+If a cluster does not have a valid or current license, the command outputs an error message.
```

Comment:
This conflicts with the intro stating we mention something about AGPLv3

Reply:
Changed to reflect that the error occurs if the commercial license is expired.